docs/actions/backup_file.md

```diff
@@ -1,6 +1,6 @@
 <!--
 This file is auto-generated and will be re-generated every time the docs are updated.
-To modify it, go to its source at https://github.com/fastlane/fastlane.
+To modify it, go to its source at https://github.com/fastlane/fastlane/blob/master/fastlane/lib/assets/ActionDetails.md.erb
```

Comment:
Well, that's not really accurate, it's actually `actions/backup_file.rb`, right? :)

Reply:
Ohhhhhhh 👀 That comment almost needs to be dynamic 😱 Will create an issue for that

Reply:
Damn, sorry!

But, can we actually make them dynamic? For example, the `upload_to_app_store` action's doc is composed of https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/upload_to_app_store.rb and https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/docs/upload_to_app_store.md.erb, so it will be a little tricky.